.github/workflows/CI.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
         run: ./scripts/gear.sh format doc --check
 
       - name: "Check clippy: Gear"
-        run: ./scripts/gear.sh clippy gear --all-targets --all-features
+        run: ./scripts/gear.sh clippy gear --all-targets --all-features --locked
```

Comment:
seems it doesn't work for cargo build, why should do here?

Reply:
`cargo clippy` without `--locked` will update `Cargo.lock`, and then the next steps will pass successfully even with `--locked` flag.

Reply:
This has been added for `check` stage breaks at the early stage too. But we can leave `--locked` in the `build` stage only indeed.